README.md

```diff
@@ -50,6 +50,10 @@ const maxInscribedCircle = require('max-inscribed-circle/dist/max-inscribed-circ
 
 From there, usage should be the same.
 
+## User Guide
+
+An in-depth user guide can be found at [doc/guide.md](doc/guide.md).
```

Comment:
You might want to add the triangle image from the guide to the top of the page, describing what this library does, just as a thought.

Reply:
I thought about that, but the only place I felt it belonged was at the very beginning and I wasn't sure if I liked having an image there. I'll give it a shot though. It'll probably grow on me.